internal/path/path.mbt

```diff
@@ -65,13 +65,26 @@ pub fn ext(path : StringView) -> StringView {
 
 ///|
 /// Extract stem part exclude `.`
-/// 
-/// property: `stem(path) + "." + ext(path)`
+///
+/// property: `stem(path) + ext(path) == path`
```

Comment:
/cc @illusory0x0 updated the property holds between the `stem` and `ext` function